README.md

```diff
@@ -31,16 +31,30 @@ Research day should be thought of as a [spike](http://www.extremeprogramming.org
 
 ## Week 4 - Node 1
 
++ [Deployment](./week-4/Deployment.md)
+
++ [Packaging](./week-4/packaging.md)
+
 ## Week 5 - Node 2
 
 ## Week 6 - Postgres
 
++ [Build Scripts](./week-66/build_scripts.md)
+
++ [Script Injections](./week-6/script-injection.md)
+
 ## Week 7 - Authentication
 
-+ [stateless vs stateful authentication](.week7/stateless_vs_stateful_authentication.md)
++ [stateless vs stateful authentication](.week-7/stateless_vs_stateful_authentication.md)
 
-+ [Attacks](./week7/Attacks.md)
++ [Attacks](./week-7/Attacks.md)
+
++ [Web Storage](./week-7/web_storage.md)
 
 ## Week 8 - Express
 
 + [Sessions](./week8/sessions.md)
+
++ [Fetch and Promises](./week8/fetch-promises-research.md)
```

Comment:
This link is not broken,
Because although you did not change the folder's name(as per week6 and week7),
You also did not change the name of the folder when you referenced it in this link(wheras you linked a week-6, week-66, and week-7, which did not exist)

Although this works, when you do a large project and hand it over to someone else, not following some sort of naming consistency would make it more difficult for the next person to continue your work.
Google "Naming Convention" or "Universal Naming Convention"

A naming consistency adds clarity and uniformity that helps everyone to remember, understand, and visually scan your code, and even for searching your code.
Some word processing programs can filter, highlight, and edit names that have certain qualities in them (eg, hyphens), which makes it significantly easier to make quick edits that would have otherwise demanded many long, complicated, edits that you'd have to search hard and carefully in order to even find

I advise changing the link to week-8 as well as changing the folder's name to week-8

Reply:
@HilbertSpitzer good point! Made some changes